### ISSUE TITLE
[SUPPORT] PYMEAcquire config doc pretty lacking

### ISSUE THREAD

**Issue by barentine** [python-microscopy/python-microscopy#1165]
We've added some metadata mandates / shifted things around a bit and I'm not sure we're keeping up with the doc.

Specifically we ought to update things after PR #1131 , 
![image](https://user-images.githubusercontent.com/31105780/145096062-4d1f46e9-63b4-4750-a166-e958be1c0b15.png)
to include exactly what one needs to to in order to start PYMEAcquire with their camera and acquire a snapshot without digging through the metadata rabbit hole

```
 Error whilst running Acquire>Snapshot	F5
==============================================
python-microscopy=21.09.23.post0.dev[git]
python=3.7.10, platform=win32
numpy=1.19.2, wx=4.0.4 msw (phoenix) wxWidgets 3.0.5

Traceback
=========
Traceback (most recent call last):
  File "c:\userfiles\code\python-microscopy\PYME\ui\progress.py", line 153, in func
    return fcn(*args, **kwargs)
  File "c:\userfiles\code\python-microscopy\PYME\Acquire\acquiremainframe.py", line 486, in OnMAquireOnePic
    mdgen(mdh)
  File "c:\userfiles\code\python-microscopy\PYME\Acquire\Hardware\Camera.py", line 784, in GenStartMetadata
    mdh.setEntry('Camera.Name', self.GetName())
  File "c:\userfiles\code\python-microscopy\PYME\Acquire\Hardware\Camera.py", line 177, in GetName
    raise NotImplementedError('Implemented in derived class.')
NotImplementedError: Implemented in derived class.

```

**Comment by David-Baddeley** [comment]
ahh, I let the `GetName()` in by accident - I'm not sure it's really useful and it should definitely fail safely. Docs could also use an overhaul

**Comment by David-Baddeley** [comment]
Low hanging fruit addressed in #1163

**Comment by barentine** [comment]
other low hanging fruit would be to give a brief description of how to configure a scope if you aren't using init script / protocols in the main repo. Quick example of calling an external init script, and description of where to save protocols so they are automatically found (`config_dir/protocols/` would be good

**Comment by barentine** [comment]
re: protocols I think it would also be nice to configure a separate protocols directory, such that one might be able to keep all of their PYME-custom scope stuff in a single repository of their own

edit: I suppose that can already be done if the .PYME directory is in version control and you put all of your protocols there

**Comment by barentine** [comment]
addressed for now with #1224 and #1163
